sid/script.js

```diff
@@ -0,0 +1,633 @@
+
+//selecting a random word from dictionary
+let random = dictionary[Math.floor(Math.random() * dictionary.length)];
+console.log(random);
+
+for (const button of allButtons) {
+    if (gameLost === false) {
+        button.addEventListener('click', wordFound);
+        button.addEventListener('click', checkFullSquares);
+    }
+}
+
+if (gameLost === false) {
+    document.addEventListener('keyup', wordFound);
+    document.addEventListener('keyup', checkFullSquares);
+    moreChances.addEventListener('click', addTwoRows);
+}
+
+restartButton.addEventListener('click', reset);
+
+function pickRandomColor() {
+    const arr = ['#ffff66', '#660066', 'pink'];
+    return arr[Math.floor(Math.random(0, 1) * 3)];
+}
+
+function changeColor() {
+    if (gameComplete === true) {
+        mainHeading.style.backgroundColor = pickRandomColor();
+    }
+}
+
+function reset() {
+    setTimeout(function () {
+        notificationOne.textContent = '';
+        notificationTwo.textContent = '';
+        notificationThree.textContent = '';
+        mainHeading.style.opacity = '0';
+        mainHeading.style.backgroundColor = '';
+        mainHeading.style.transitionDuration = '1s';
+        mainHeading.style.transitionTimingFunction = 'ease-in';
+        document.body.style.backgroundImage = 'none';
+        document.body.style.transitionDuration = '1s';
+        document.body.style.transitionTimingFunction = 'ease-in';
```

Comment:
It can be cumbersome to update the styles in JS, see if you can use [classList.toggle()](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList) to switch styles.